src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/FileStorageService.cs

```diff
@@ -23,13 +22,11 @@ public FileStorageService(IBlobStorageService blobStorageService)
             _blobStorageService = blobStorageService;
         }
 
-        public async Task<SubjectData> GetSubjectData(ImportMessage importMessage)
+        public async Task<SubjectData> GetSubjectData(Guid releaseId, string observationsFileBlobPath)
```

Comment:
Do you think `FileStorageService` could be eliminated now and shift the little logic that there is into the calling service? It's largely been replaced by the common BlobStorageService now and it's been a recent intention to get rid of the various FileStorageServices throught the project.

Also this class has an unused method `StreamFile`

Reply:
Created https://dfedigital.atlassian.net/browse/EES-1743